frontend/src/app/supportAdmin/ManageUsers/UserSearch.tsx

```diff
@@ -0,0 +1,63 @@
+import { faSlidersH } from "@fortawesome/free-solid-svg-icons";
+import React from "react";
+
+import SupportHomeLink from "../SupportHomeLink";
+import Button from "../../commonComponents/Button/Button";
+import SearchInput from "../../testQueue/addToQueue/SearchInput";
+
+interface UserSearchProps {
+  onClearFilter: () => void;
+  onSearchClick: (e: any) => void;
+  onInputChange: (e: any) => void;
+  searchEmail: string;
+  disableClearFilters: boolean;
+}
+export const UserSearch: React.FC<UserSearchProps> = ({
+  onClearFilter,
+  onSearchClick,
+  onInputChange,
+  searchEmail,
+  disableClearFilters,
+}) => {
+  return (
+    <div className="prime-container card-container padding-bottom-3">
+      <div className="usa-card__header">
+        <div className="width-full">
+          <SupportHomeLink />
+          <div className="grid-row width-full margin-top-1">
+            <h1 className="desktop:grid-col-fill tablet:grid-col-fill mobile:grid-col-12 font-heading-lg margin-bottom-0">
+              Manage user
```

Comment:
based on laura's comment here https://github.com/CDCgov/prime-simplereport/issues/6117#issuecomment-1644291485 -- looks like we want this page to be `Manage users` plural